python-checks/src/test/resources/checks/methodShouldBeStatic.py

```diff
@@ -133,6 +133,37 @@ def fun31(slf): raise ValueError('Some error') # Noncompliant
     def fun25(slf, other):
         # some comment
         raise NotImplementedError
-                
+
+    # from here on, we test built-in functions
+
+    def __init__(slf):
+        print(1)
+
+    def __init__(slf):
+        print(slf)
+
+    def __this_is_not_a_built_in_method__(slf):
```

Comment:
@ivandalbosco why such name? (as i understood your implementation considers it as a built-in method)